src/main/java/com/minecolonies/entity/ai/minimal/EntityAIGateInteract.java

```diff
@@ -82,33 +83,33 @@ public EntityAIGateInteract(@NotNull EntityLiving entityIn)
     @Override
     public boolean shouldExecute()
     {
-        return this.theEntity.isCollidedHorizontally && checkPathEntity();
+        return this.theEntity.isCollidedHorizontally && checkPath();
     }
 
     /**
      * Checks if there exists a path..
      *
      * @return true if the fence gate can be passed.
      */
-    private boolean checkPathEntity()
+    private boolean checkPath()
     {
         @NotNull PathNavigateGround pathnavigateground = (PathNavigateGround) this.theEntity.getNavigator();
-        PathEntity pathentity = pathnavigateground.getPath();
-        return pathentity != null && !pathentity.isFinished() && pathnavigateground.getEnterDoors() && checkFenceGate(pathentity);
+        Path Path = pathnavigateground.getPath();
+        return Path != null && !Path.isFinished() && pathnavigateground.getEnterDoors() && checkFenceGate(Path);
     }
 
     /**
      * Checks if the citizen is close enough to an existing fence gate.
      *
-     * @param pathentity the path through the fence.
+     * @param Path the path through the fence.
      * @return true if the gate can be passed
      */
-    private boolean checkFenceGate(@NotNull PathEntity pathentity)
+    private boolean checkFenceGate(@NotNull Path Path)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)